### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by dveeden**
Add a template for pull requests

This is BSD licensed.

### CODE REVIEW

**Comment by percona-csalguero** [comment]
could you fix the conflicts please?

**Comment by percona-csalguero** [comment]
@dveeden Sorry for the delay. Could you fix the conflicts to merge this commit?
Thanks

**Comment by dveeden** [comment]
@percona-csalguero done

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- [ ] The contributed code is licensed under BSD
```
Comment (percona-csalguero) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] The contributed code is licensed under GPL v2.0
```
https://github.com/percona/percona-toolkit/blob/3.x/COPYING

**Review by percona-csalguero — CHANGES_REQUESTED** [review]

**Review by svetasmirnova — CHANGES_REQUESTED** [review]
We updated CONTRIBUTE.md and CONTRIBUTING.md in the current main branch 3.x. Renaming CONTRIBUTE.md to CONTRIBUTING.md wipes out these changes. Could you please remove this file from this PR?

**Comment by dveeden** [comment]
@svetasmirnova done. PTAL.

**Review by percona-csalguero — APPROVED** [review]

**Review by svetasmirnova — APPROVED** [review]

**Pull request merged**